Spec/PushAdmin.swift

```diff
@@ -542,13 +542,30 @@ class PushAdmin : QuickSpec {
                     ]
 
                     waitUntil(timeout: testTimeout) { done in
+                        realtime.push.admin.channelSubscriptions.list(params) { result, error in
+                            guard let result = result else {
+                                fail("PaginatedResult should not be empty"); done(); return
+                            }
+                            expect(result.items).to(contain(expectedRemoved))
+                            expect(error).to(beNil())
+                            done()
+                        }
+                    }
+
+                    waitUntil(timeout: testTimeout) { done in
                         realtime.push.admin.channelSubscriptions.removeWhere(params) { error in
                             expect(error).to(beNil())
                             done()
                         }
                     }
 
                     waitUntil(timeout: testTimeout) { done in
+                        delay(3) {
```

Comment:
@tcard This test only passes if I put a delay between the `removeWhere:params` call and the `list:params` call. Without a delay the `list:params` returns 2 subscriptions instead of 0. It seems the backend needs some time to "commit" the changes.

Reply:
@ricardopereira Are you setting `fullWait=true`? Sorry, I should've mentioned and it's not in the spec. I set up [a `pushFullWait: bool` client option](https://github.com/ably/ably-java/pull/355/files#diff-f111f958256bd5375e3c22f2b1fed74bR182) and then, if it's on, [add a `fullWait=true` query parameter to every push request](https://github.com/ably/ably-java/pull/355/files#diff-7006bd95b380da0538f1ca6a1971e2b2R76), so that it waits for the full operation to complete. (We don't do that by default because it can be potentially a quite expensive operation.)

Reply:
@tcard Thanks. It works now as expected 😄